catalog/ui/catalog-ui-search/src/main/webapp/component/sort-item/sort-item.view.js

```diff
@@ -177,11 +177,11 @@ define([
             });
 
         },
-        getValue: function () {
-            return {
-                sortField: this.sortAttribute.currentView.model.getValue()[0],
-                sortOrder: this.sortDirection.currentView.model.getValue()[0]
-            }
+        getSortField: function () {
+            return this.sortAttribute.currentView.model.getValue()[0];
+        },
+        getSortOrder: function () {
```

Comment:
Since these are changed it looks like the `getValue` method was also used in `query-alititude.view` as well so that will need to be updated.

Reply:
:+1:
I updated the `saveToModel` method in `query-altitude.view` to:
```javascript
    // matches saveToModel in query-settings.view
    saveToModel: function () {
        var federation = this.settingsFederation.currentView.model.getValue()[0];
        var src;
        if (federation === 'selected') {
            src = this._srcDropdownModel.get('value');
            if (src === undefined || src.length === 0) {
                federation = 'local';
            }
        }
        var sortField = this.settingsSortField.currentView.getSortField();
        var sortOrder = this.settingsSortField.currentView.getSortOrder();
        this.model.set({
            src: src,
            federation: federation,
            sortField: sortField,
            sortOrder: sortOrder
        });
    },
```
Note that I also added the `federation = 'local';` step, which was not in `query-altitude.view` before.

Also, I updated the test steps to include your suggestion.

Reply:
I think you can get rid of that `saveToModel` method in `query-altitude.view`.  Looks like it isn't used (`construct filters` is).

Reply:
@andrewkfiedler, I added a commit to remove that method. Let me know if this PR should be reheroed.

Reply:
Nah, it should be fine.  Thanks.